.storybook/config.js

```diff
@@ -30,7 +30,7 @@ addParameters({
     }),
   },
   backgrounds: [
-    {name: 'Sky Light', value: tokens.colorSkyLight, default: true},
+    // {name: 'Sky Light', value: tokens.colorSkyLight, default: true},
```

Comment:
We'll need to figure out how to handle background color in Storybook.

Reply:
We should also have a way to tell storybook not to render an `AppProvider`, since theme examples bring their own `AppProvider`. This way we avoid doubling up.

Reply:
This stuff is using the backgrounds addon: https://www.npmjs.com/package/@storybook/addon-backgrounds

Currently we configure bits here then we unset the body background color that polaris sets down here: https://github.com/Shopify/polaris-react/blob/master/.storybook/preview-head.html#L2-L9 It looks like that can be removed now you're setting the background color on the ThemeProvider instead of the body.

We might be able to do some clever things around their on event API. We could improve our AppProvider decorator so that if a background change is detected then we mash a new theme config into our AppProvider.

Our AppProvider decorator is  at https://github.com/Shopify/polaris-react/blob/master/.storybook/config.js#L46-L52

Changing that to be

```

addDecorator(function AppProviderDecorator(story) {
  React.useEffect(() => {
    window.__STORYBOOK_ADDONS_CHANNEL__.on(
      'storybook/background/update',
      (bg) => {
        console.log('Background color', bg.selected);
        console.log('Background name', bg.name);
      },
    );
  }, []);

  return (
    <div style={{padding: '8px'}}>
      <AppProvider i18n={en}>{story()}</AppProvider>
    </div>
  );
});
```

gives me some console logs - form there you can build out a theme object based on the selected name and jam it into the app provider

---

Currently that is a global decorator, we could remove that and instead add it explictly to each story that needs it (within polaris-readme-loader.js), and then have some kind of whitelist that doesn't add the AppProvider wrapper to certain storeis